packages/ui-library/src/theme/typography.ts

```diff
@@ -197,4 +197,12 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     lineHeight: 1.33, // 16/12
     color: colors[mode].typography.typo3,
   },
+  tablePaginationLabel: {
+    fontFamily: 'Archivo',
```

Comment:
we should not be using other fontFamilies, check with designers if they have missed this, but it should all be Inter always